app/controllers/api/v1/subjects_controller.rb

```diff
@@ -75,7 +75,7 @@ def selector
     @selector ||= Subjects::Selector.new(api_user.user,
       workflow,
       params,
-      controlled_resources)
+      Subject.all)
```

Comment:
Do we need to pass a scope at all here? I think we can remove this from the interface.